### PR TITLE
Update translation files

### PR DESCRIPTION
**Pull request by ihhub**
We made some changes in text recently.

### CODE REVIEW

File: files/lang/ru.po [hunk 1]
```diff
@@ -1849,6 +1878,46 @@ msgstr "Выход"
...
+msgid "Settings"
+msgstr "Настройки"
+
+#, fuzzy
+msgid "Select Game Language"
+msgstr "Выберите язык игры:"
+
+#, fuzzy
+msgid "Change language of the game."
+msgstr "Изменить разрешение экрана."
```
Comment (a1exsh) [inline files/lang/ru.po:1901]:
```suggestion
msgstr "Изменить язык интерфейса."
```
?

File: files/lang/ru.po [hunk 2]
```diff
@@ -1849,6 +1878,46 @@ msgstr "Выход"
...
+
+msgid "Change resolution of the game."
+msgstr "Изменить разрешение экрана."
+
+msgid "Experimental game settings."
+msgstr "Экспериментальные настройки игры."
+
+#, fuzzy
+msgid "Attention"
+msgstr "Присоединение"
```
Comment (a1exsh) [inline files/lang/ru.po:1914]:
```suggestion
msgstr "Внимание"
```

File: files/lang/ru.po [hunk 3]
```diff
@@ -124,14 +124,13 @@ msgstr "Тысячи"
...
-msgstr "Отряды 4-х рас -2"
+#, fuzzy
+msgid "Multiple"
+msgstr "Мульти"
 
-msgid "Troops of 5 alignments -3"
-msgstr "Отряды 5-и рас -3"
+#, fuzzy
+msgid "Troops of %{count} alignments -%{penalty}"
+msgstr "Отряды 3-х рас -1"
```
Comment (qarkai) [inline files/lang/ru.po:133]:
```suggestion
msgstr "Отряды %{count} рас -%{penalty}"
```

File: files/lang/ru.po [hunk 4]
```diff
@@ -541,6 +546,12 @@ msgid ""
 msgstr ""
 "Благодаря орлиному взору, %{name} смог выучить заклинание \"%{spell}\"."
 
+msgid "Monsters"
+msgstr "Монстры"
+
+msgid "N/A"
+msgstr ""
```
Comment (qarkai) [inline files/lang/ru.po:553]:
```suggestion
msgstr "Н/Д"
```

File: files/lang/ru.po [hunk 5]
```diff
@@ -1714,18 +1728,33 @@ msgstr "неделя"
...
 
+msgid "Puzzle"
+msgstr ""
+
 msgid "View the obelisk puzzle."
 msgstr "Показать мозаику."
 
+#, fuzzy
+msgid "Scenario Information"
+msgstr "Разомкнутый строй"
```
Comment (qarkai) [inline files/lang/ru.po:1746]:
```suggestion
msgstr "Информация о сценарии"
```

File: files/lang/ru.po [hunk 6]
```diff
@@ -1849,6 +1878,46 @@ msgstr "Выход"
 msgid "Quit out of Free Heroes of Might and Magic II."
 msgstr "Выйдите из Free Heroes of Might and Magic II."
 
+msgid "Language"
+msgstr ""
```
Comment (qarkai) [inline files/lang/ru.po:1882]:
```suggestion
msgstr "Язык"
```

File: files/lang/ru.po [hunk 7]
```diff
@@ -1849,6 +1878,46 @@ msgstr "Выход"
...
+
+#, fuzzy
+msgid "Resolution"
+msgstr "Революция"
```
Comment (qarkai) [inline files/lang/ru.po:1886]:
```suggestion
msgstr "Разрешение"
```

File: files/lang/ru.po [hunk 8]
```diff
@@ -1849,6 +1878,46 @@ msgstr "Выход"
...
+
+#, fuzzy
+msgid "Experimental"
+msgstr "Элементал"
```
Comment (qarkai) [inline files/lang/ru.po:1890]:
```suggestion
msgstr "Экспериментальные"
```

File: files/lang/ru.po [hunk 9]
```diff
@@ -1889,12 +1958,32 @@ msgstr ""
...
+
+msgid "Your Funds"
+msgstr ""
+
+msgid "Planned Gift"
+msgstr ""
+
+#, fuzzy
+msgid "Gift from %{name}"
+msgstr "Нанять  %{name}"
```
Comment (qarkai) [inline files/lang/ru.po:1985]:
```suggestion
msgstr "Дар от %{name}"
```

File: files/lang/ru.po [hunk 10]
```diff
@@ -1849,6 +1878,46 @@ msgstr "Выход"
...
+#, fuzzy
+msgid "Select Game Language"
+msgstr "Выберите язык игры:"
+
+#, fuzzy
+msgid "Change language of the game."
+msgstr "Изменить разрешение экрана."
+
+msgid "Select Game Resolution"
+msgstr "Выбор Разрешения Экрана"
```
Comment (qarkai) [inline files/lang/ru.po:1904]:
```suggestion
msgstr "Выбор разрешения экрана"
```

**Pull request merged**